tests/manage/pv_services/test_pvc_delete_subvolumegroup.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import logging
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pvc import delete_pvcs
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+from ocs_ci.framework.testlib import ManageTest, skipif_ocs_version, polarion_id
+from ocs_ci.utility.utils import TimeoutSampler
+
+logger = logging.getLogger(__name__)
+
+
+@skipif_ocs_version("<4.12")
+@polarion_id("OCS-4664")
+class TestPvcDeleteSubVolumeGroup(ManageTest):
+    """
+    Delete PVC subvolumegroup and make sure PVC reaches bound state
+    """
+
+    subvolumegroup_ls_cmd = (
+        "ceph fs subvolumegroup ls ocs-storagecluster-cephfilesystem"
```

Comment:
This command may not work on an external mode cluster as the volume name would be different. 
I think it is `cephfs` please check and make the changes accordingly.

Reply:
Done.